tests/unit/test_distro.py

```diff
@@ -471,10 +429,20 @@ def test_configure_no_env(self, mocker):
         mocker.patch.dict(os.environ, {})
         distro.SolarWindsDistro()._configure()
         assert os.environ[OTEL_PROPAGATORS] == "tracecontext,baggage,solarwinds_propagator"
-        assert os.environ[OTEL_TRACES_EXPORTER] == "solarwinds_exporter"
+        assert os.environ[OTEL_TRACES_EXPORTER] == "otlp_proto_http"
```

Comment:
See [this comment](https://github.com/solarwinds/apm-python/pull/499#discussion_r1937722619)